debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java

```diff
@@ -83,7 +84,7 @@ public PostgresStreamingChangeEventSource(PostgresConnectorConfig connectorConfi
         this.errorHandler = errorHandler;
         this.clock = clock;
         this.schema = schema;
-        pauseNoMessage = DelayStrategy.constant(taskContext.getConfig().getPollInterval().toMillis());
+        pauseNoMessage = DelayStrategy.constant(Duration.ofMillis(taskContext.getConfig().getPollInterval().toMillis()));
```

Comment:
`constant` takes the Duration so there is no need for conversion to milliseconds and back to Duration.